packages/react-meteor-data/useFind.ts

```diff
@@ -71,6 +71,24 @@ const fetchData = <T>(cursor: Mongo.Cursor<T>) => {
   return data
 }
 
+const useSyncEffect = (effect, deps) => {
+  const [cleanup, timeoutId] = useMemo(
+    () => {
+      const cleanup = effect();
```

Comment:
Deduplicate cleanup calls, GC cleanup function reference, and handle `undefined` cleanup function:
```suggestion
      let _cleanup = effect();
      const cleanup = () => {
        if (_cleanup === undefined) return;
        _cleanup();
        _cleanup = undefined;
      };
```